recipes/suitesparse/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+if [ "$(uname)" == "Darwin" ]
+then
+    export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+    cp -f "${RECIPE_DIR}/SuiteSparse_config_Mac.mk" SuiteSparse_config/SuiteSparse_config.mk
+else
+    export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+    cp -f "${RECIPE_DIR}/SuiteSparse_config_linux.mk" SuiteSparse_config/SuiteSparse_config.mk
+fi
+
+
+export INCLUDE_PATH="${PREFIX}/include"
+export LIBRARY_PATH="${PREFIX}/lib"
+
+export INSTALL_LIB="${PREFIX}/lib"
+export INSTALL_INCLUDE="${PREFIX}/include"
+
+export BLAS="-lopenblas"
+export LAPACK="-lopenblas"
```

Comment:
I really want to stay away from OpenMP for SuiteSparse. So many things are use to SuiteSparse's static linking and it is unclear to me what strategy SuiteSparse takes with statically linking to other libraries.

We know that statically linking to GOMP ends up being a horrible plan with breakage written all over. See this `gcc` [bug report]( https://gcc.gnu.org/bugzilla/show_bug.cgi?id=39176#c2 ) for details on why this is a bad idea. So, I'm eager to stay away from GOMP here because of this problem. Note GOMP is how we support OpenMP on Mac and Linux.